Makefile

```diff
@@ -1,30 +1,30 @@
-GOLANGCI_LINT_VERSION = $(shell head -n 1 .golangci.yml | tr -d '\# ')
```

Comment:
I see that, for some reason, the change affected whole formatting. Is it possible to keep only the actual change(s)?

Reply:
Sure I will make the formatting correct.

Reply:
Hi @vanshaj 

It seems like the whole file formatting is still affected. It shows that 30 lines were removed and 30 added:
![image](https://user-images.githubusercontent.com/5425600/180960806-7f663a1c-30a2-4fe0-9305-9f9a9c252d87.png)